core/CometsLayout.m

```diff
@@ -30,6 +30,9 @@
         initial_pop = [0];%models by x by y, default 0. Units = grams.
         external_rxns = table();
         external_rxn_mets = table();
+        periodic_media_mode = 'none';

```

Comment:
periodic_media_mode should be added to the CometsParams class, with some default (or 'none'). Let's also verify the implementation in Java follows the conventions for Global Parameters

Reply:
It is currently not a parameter, only a flag used to distinguish between the two modes. It is possible to rewrite the code and avoid this flag completely, however I think that would reduce the readability/ease of interpretation of the code.

Reply:
So it's not supposed to be written in the list of params? There's a line in writeCometsLayout (line ~104) that flags parameters to not be printed. 

The general idea was that everything not related to the individual contents of the layout would be params, so you could conceivably create your own sets of defaults and easily swap them in and out. Flags that don't refer to any specific models/metabolites/locations don't need to live in the Layout struct itself. See CometsParams.objectiveStyle, CometsParams.biomassMotionStyle. If we put this kind of flag here it might just move the readability issue here from CometsParams.

Reply:
Removed the flag.